Pod/Classes/RealexComponent/HPPManager.swift

```diff
@@ -15,8 +15,54 @@ import UIKit
 
 }
 
+/**
+ *  The delegate callbacks which allow the host app to receive all possible results from the component using a generic decodable type.
+ */
+public protocol GenericHPPManagerDelegate: class {
+    
+    associatedtype PaymentServiceResponse: Decodable
+    func HPPManagerCompletedWithResult(_ result: PaymentServiceResponse)
+    func HPPManagerFailedWithError(_ error: Error?)
+    func HPPManagerCancelled()
+    
+}
+
+/**
+ *  A type-erased implementer of the `HPPSwiftManagerDelegate` protocol
```

Comment:
Are we able to update this comment to reflect the name change?

Reply:
Fixed!